### DISCUSSION TITLE
Q

### DISCUSSION THREAD

**Discussion by tperchoc** [pnnl/OpenCGRA#16] [Q&A] [answered]
Good morning, 
I wanted to know if I could find documentation regarding the use of this OpenCGRA project (for example what file corresponds to, how to use the different modules etc.)? I saw the publications regarding this project but unfortunately I didn't really find how to use it in practice.

Thanks in advance, 
Thibault

**Comment by tancheng** [comment, marked as answer]
Plz try this one: https://github.com/tancheng/CGRA-Flow
